packages/block-library/src/navigation-link/index.php

```diff
@@ -418,54 +455,76 @@ function block_core_navigation_link_build_variations() {
 }
 
 /**
- * Register the navigation link block.
+ * Registers the navigation link block.
  *
- * @uses render_block_core_navigation()
+ * @uses render_block_core_navigation_link()
+ * @uses build_navigation_link_block_variations()
  * @throws WP_Error An WP_Error exception parsing the block definition.
  */
 function register_block_core_navigation_link() {
+	/*
+	 * On Core versions < 6.5 the get_block_type_variations is not available.
+	 * Therefore directly register the (until here) known variations.
+	 * This keeps pre-6.5 behaviour in pre-6.5 versions.
```

Comment:
```suggestion
	 * Therefore directly register the variations for post/taxonomies that have been registered at this point.
	 * This keeps pre-6.5 behaviour in pre-6.5 versions.
```

I think this improves the comment.